docs/docsite/rst/community/maintainers_workflow.rst

```diff
@@ -89,7 +89,7 @@ If you feel you don't have time to maintain your collection anymore, you should:
 - If the collection is under the ``ansible-collections`` organization, also inform relevant :ref:`communication_irc`, the ``community`` chat channels on IRC or matrix, or by email ``ansible-community@redhat.com``.
 - Look at active contributors in the collection to find new maintainers among them. Discuss the potential candidates with other maintainers or with the community team.
 - If you failed to find a replacement, create a pinned issue in the collection, announcing that the collection needs new maintainers.
-- Make the same announcement through the `Bullhorn newsletter <https://github.com/ansible/community/wiki/News#the-bullhorn>`_.
+- Make the same announcement through the `Bullhorn newsletter <https://forum.ansible.com/c/news/bullhorn/17>`_.
```

Comment:
```suggestion
- Make the same announcement through the `Bullhorn newsletter <https://forum.ansible.com/c/news/bullhorn/17>`_.
```
@cybette should we add a link to how to post an announcement in Bullhorn in the description right after the subscription link?

Reply:
hmm I thought it will take the whole ["about" topic](https://forum.ansible.com/t/about-the-newsletter-category/166) which includes how to post to Bullhorn, but I guess it's only the first paragraph.

We can either
- combine the 2 paragraphs into one
- link to [About the Newsletter category](https://forum.ansible.com/t/about-the-newsletter-category/166) instead 

@GregSutcliffe any recommendations?

Reply:
I can't find a preview to look at, but from a naive perspective, I think either could work. I lean slightly towards linking to the category because that paragraph gets a bit big otherwise, but I'd be fine with either option.

Reply:
@cybette @GregSutcliffe thanks! @samccann could you please adjust the PR correspondingly?